.github/workflows/CI-lint-format.yml

```diff
@@ -54,6 +55,10 @@ jobs:
         shell: bash
         run: ${{ env.DOCKER_COMPOSE_CMD }} cargo fmt --check 2>&1 | tee formatting_output.txt
 
+      - name: Features
+        shell: bash
+        run: ${{ env.DOCKER_COMPOSE_CMD }} zepter run --config .cargo/zepter-ci.yaml | tee features_output.txt
```

Comment:
Do we really need two separate config files for zepter ?

Reply:
I'm afraid that's the best tradeoff. It looks like zepter is not really flexible in terms of cli, so that if you use a config file, you cannot extend the execution flags that from the command line. Given that we want different flags for the CI run and for the local run, and I want to keep the local usage as simple as executing `zepter` in the project root, I believe that having separate config files is the best solution.
An alternative would be to encode all the execution flags in the workflow yml, but I think that's less readable and maintainable.